src/goInstallTools.ts

```diff
@@ -355,11 +355,22 @@ export async function promptForUpdatingTool(toolName: string) {
 	}
 	const goVersion = await getGoVersion();
 	const updateMsg = `Your version of ${tool.name} appears to be out of date. Please update for an improved experience.`;
-	vscode.window.showInformationMessage(updateMsg, 'Update').then((selected) => {
+	var choices: string[] = ['Update'];
+	if (toolName == `gopls`) {
+		choices.push('Release Note');  // TODO(hyangah): pass more info such as version, release note location.
```

Comment:
I think this should probably be "Release Notes" - I checked, and when there is a new version of VS Code, the title of the page that pops up is "Release Notes: <version>".

Reply:
I agree

Reply:
done.